md/builders.py

```diff
@@ -1,15 +1,59 @@
+from tempfile import NamedTemporaryFile
+
 import numpy as np
 from simtk import unit
 from simtk.openmm import app, Vec3
 
+from openeye.oechem import (
+    oemolistream,
```

Comment:
My first time seeing this kind of import setup - pretty nice!

Reply:
I much prefer explicit imports to module imports, unless absolutely necesary